Traceback (most recent call last):
  File "c:\pythin35\python35\lib\tokenize.py", line 390, in find_cookie
    line_string = line.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 2: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\pythin35\python35\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\pythin35\python35\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Pythin35\Python35\Scripts\pylint.exe\__main__.py", line 9, in <module>
  File "c:\pythin35\python35\lib\site-packages\pylint\__init__.py", line 19, in run_pylint
    Run(sys.argv[1:])
  File "c:\pythin35\python35\lib\site-packages\pylint\lint.py", line 1394, in __init__
    linter.check(args)
  File "c:\pythin35\python35\lib\site-packages\pylint\lint.py", line 801, in check
    self._do_check(files_or_modules)
  File "c:\pythin35\python35\lib\site-packages\pylint\lint.py", line 920, in _do_check
    for descr in self.expand_files(files_or_modules):
  File "c:\pythin35\python35\lib\site-packages\pylint\lint.py", line 952, in expand_files
    self.config.black_list_re)
  File "c:\pythin35\python35\lib\site-packages\pylint\utils.py", line 1062, in expand_modules
    spec = modutils.file_info_from_modpath(modparts, path=sys.path)
  File "c:\pythin35\python35\lib\site-packages\astroid\modutils.py", line 427, in file_info_from_modpath
    return _spec_from_modpath(modpath, path, context)
  File "c:\pythin35\python35\lib\site-packages\astroid\modutils.py", line 649, in _spec_from_modpath
    found_spec = spec.find_spec(modpath, path)
  File "c:\pythin35\python35\lib\site-packages\astroid\interpreter\_import\spec.py", line 292, in find_spec
    submodule_path or path)
  File "c:\pythin35\python35\lib\site-packages\astroid\interpreter\_import\spec.py", line 254, in _find_spec_with_path
    spec = finder.find_module(modname, module_parts, processed, submodule_path)
  File "c:\pythin35\python35\lib\site-packages\astroid\interpreter\_import\spec.py", line 100, in find_module
    stream, mp_filename, mp_desc = imp.find_module(modname, submodule_path)
  File "c:\pythin35\python35\lib\imp.py", line 302, in find_module
    encoding = tokenize.detect_encoding(file.readline)[0]
  File "c:\pythin35\python35\lib\tokenize.py", line 431, in detect_encoding
    encoding = find_cookie(first)
  File "c:\pythin35\python35\lib\tokenize.py", line 395, in find_cookie
    raise SyntaxError(msg)
SyntaxError: invalid or missing encoding declaration for 'C:\\InformationSystem\\myFiles\\queens.py'
